projects/mac_ai/testing/prepare_release.py

```
@@ -0,0 +1,224 @@
+import os
+import pathlib
+import logging
+import tarfile
+
+from projects.core.library import env, config, run, configure_logging, export
```

Ruff (0.11.9)

`projects.core.library.run` imported but unused

Remove unused import

(F401)